exercises/practice/minesweeper/.docs/instructions.md

```diff
@@ -35,3 +35,14 @@ And your code will transform it into this:
 ·2*2·
 ·111·
 ```
+
+## Performance Hint
+
+All the inputs and outputs are in Ascii. Rust `String`s and `&str` are utf8,
+so while one might expect "Hello".chars() to be simple, it actually has to
+check each char to see if it's 1, 2, 3 or 4 `u8`s long. If we know a `&str`
+is ascii then we can call `.as_bytes()` and turn it into a `&[u8]` slice.
```

Comment:
We're not changing the underlying data so maybe word it as:
```suggestion
is ascii then we can call `.as_bytes()` and refer to the underlying data via a `&[u8]` slice.
```

Reply:
This is clearer, yes.